scripts/load-backup.sh

```diff
@@ -80,4 +49,4 @@ docker exec osrd-postgres pg_restore --if-exists -c -d osrd -x //tmp/backup-osrd
 echo "Analyze for performances..."
 docker exec osrd-postgres psql -d osrd -c 'ANALYZE;' > /dev/null
 
-echo Done !
+echo Load backup done !
```

Comment:
nit: you may want to use quotes to be safe/consistent

Reply:
Done in https://github.com/osrd-project/osrd/pull/6365/commits/17ddc57589f3747c161934c5447ddd075a065472